background-remover/background-remover.py

```diff
@@ -0,0 +1,20 @@
+import os
+import rembg
+
+def process (input_path: str, output_path: str, session = rembg.new_session()):
```

Comment:
what is the purpose of passing a rembg session as an argument to this function?

Reply:
i have fixed it now